src/errors.js

```diff
@@ -0,0 +1,47 @@
+var util = require('util');
+
+/**
+ * @module errors
+ */
+var errors = (module.exports = {});
+
+/**
+ * Given a response request error, sanitize sensitive data.
+ *
+ * @method    sanitizeErrorRequestData
+ * @memberOf  module:errors
+ */
+errors.sanitizeErrorRequestData = function(error) {
+  if (!error.response || !error.response.request || !error.response.request._data) {
+    return error;
+  }
+
+  Object.keys(error.response.request._data).forEach(function(key) {
+    if (key.toLowerCase().match('password|secret')) {
+      error.response.request._data[key] = '[FILTERED]';
```

Comment:
I wonder if this should say `[SANITIZED]` to better match the name of the function/feature?